build/BranchInfo.props

```diff
@@ -21,14 +21,14 @@
     <IsStableProject Condition="'$(UseStableVersionForNativeAssets)' == 'true'">true</IsStableProject>
   </PropertyGroup>
   <PropertyGroup Condition="'$(IsStableProject)' == 'true'">
-    <MajorVersion>0</MajorVersion>
-    <MinorVersion>12</MinorVersion>
+    <MajorVersion>1</MajorVersion>
+    <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <PreReleaseLabel>preview</PreReleaseLabel>
   </PropertyGroup>
   <PropertyGroup Condition="'$(IsStableProject)' != 'true'">
     <MajorVersion>0</MajorVersion>
-    <MinorVersion>12</MinorVersion>
+    <MinorVersion>13</MinorVersion>
```

Comment:
I believe we can keep this at `0.12.0`.  Master has already moved to `0.13.0`.

https://github.com/dotnet/machinelearning/blob/e5cbca78683e3c1036966e07b7fd7d1b180c1562/build/BranchInfo.props#L29-L34

Reply:
Good thing we have PRs! ;)